corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less

```diff
@@ -94,6 +94,8 @@
 @btn-footer-bg:     #474747;
 @btn-footer-border: #cccccc;
 
+@checkbox-default-color: @call-to-action-mid;
```

Comment:
Is this commit going to turn all checkboxes in HQ purple? That's be great...I tried implementing that at one point and wasn't successful.

Reply:
I wasn't reading the styles closely, missed the `form-check` class. Is there any way to make these styles more generic, maybe by tying them to the `.checkbox` class that we already use? Although it looks like the markup is a little different. Should people start exclusively using `.form-check` for checkboxes?

Reply:
@orangejenny it mimics the bootstrap 4 checkbox markup and class name...
https://v4-alpha.getbootstrap.com/components/forms/#checkboxes-and-radios

I wanted to keep the `checkbox` class as is because it requires a different HTML markup